docs/README.md

```diff
@@ -1,28 +1,116 @@
-# README
+# Documentation for Unity Catalog
 
-## Serving the documentation with mkdocs
+Unity Catalog uses [**MkDocs**](https://www.mkdocs.org/) to manage and serve its documentation. MkDocs is a simple, static site generator that’s geared towards project documentation and is written in Python. This guide will help you set up the local environment, serve the documentation locally, and deploy it to GitHub Pages.
 
-Create a virtual environment:
+## Setting Up MkDocs for Local Development
 
-```sh
+To start testing or modifying the documentation locally, follow these steps:
+
+### 1. Install MkDocs
+
+Make sure you have **MkDocs** installed in your environment. If it’s not installed, you can do so using `pip`.
+
+```bash
+pip install mkdocs
+```
+
+You can verify that it is installed by running:
+
+```bash
+mkdocs --version
+```
+
+### 2. Clone the Repository
+
+Clone the Unity Catalog repository and navigate to the documentation directory.
+
+```bash
+git clone https://github.com/unitycatalog/unitycatalog.git
+cd unitycatalog/docs
```

Comment:
Switching into the `docs` directory is not necessary. It could to confusions with step 3, as `requirements-docs.txt` is located in the top-level directory.